.github/actions/lighthouse-to-s3/action.yml

```diff
@@ -23,23 +17,20 @@ runs:
 
     - name: "Setup AWS context"
       run: |
-        sudo apt update -y \
-          && sudo apt install -y unzip \
-          && curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \ 
-          && unzip awscliv2.zip \
-          &&  sudo ./aws/install \
-          &&  rm -rf awscliv2.zip aws \
-          && export AWS_ACCESS_KEY_ID=${{ inputs.aws_access_key }} \
-          && export AWS_SECRET_ACCESS_KEY=${{ inputs.aws_secret_key }} \
-          && export AWS_DEFAULT_REGION=eu-west-1
-        if [[ "$GITHUB_REF_NAME" != "develop" || "$GITHUB_REF_NAME" != "master" ]]; then
-          VERSION=$(echo $GITHUB_REF_NAME | grep -oP '([0-9]{2}\.[0-9]{2})').x
+        sudo apt update -y 1>&-  
```

Comment:
why closing the file descriptor while it has not been opened ?

Reply:
To clear the log